.github/workflows/continuous-delivery.yml

```diff
@@ -498,7 +516,7 @@ jobs:
     runs-on: ubuntu-20.04
     env:
       # 'TEST_DEPTH' determines the matrix of E2E test cases that will be run.
-      TEST_DEPTH: ${{ needs.evaluate_options.outputs.test_level }}
+      TEST_DEPTH: ${{ needs.evaluate_options.outputs.depth }}
```

Comment:
this it's test_level not depth, yeah I know that that confusing :P

Reply:
Aha, OK, I see, we use different in python, Let me change back

Reply:
seems from here, got it
```
// testDepthEnvVarName is the environment variable we expect the user to set
// to change the default test depth level
const testDepthEnvVarName = "TEST_DEPTH"
```

Reply:
Fixed